doc/indices.sgm

```diff
@@ -90,11 +90,12 @@
 <![CDATA[CREATE INDEX test_pos_idx USING BRIN ON test (pos);]]>
           </programlisting>
           <para>
-            By default, BRINs summarize block of 128 pages. The lower numbers
-            of pages are specified, the higher granularity is reached during
-            the searches, and performance's gap between GiST indexes and BRINs
-            is lower (consider that BRINs size increases as well). Different
-            summarizations can be used with the following command:
+            By default, BRINs summarize blocks of 128 pages. The smaller the
```

Comment:
I would also propose to change BRINs to 'BRIN indexes' !?

Reply:
Yeah, the PostgreSQL documentation uses "BRIN indexes", so I think that would be correct.